src/mono/wasm/runtime/managed-exports.ts

```diff
@@ -156,10 +156,15 @@ export function init_managed_exports(): void {
     };
 
     if (MonoWasmThreads && install_sync_context) {
-        runtimeHelpers.javaScriptExports.install_synchronization_context = () => {
+        runtimeHelpers.javaScriptExports.install_synchronization_context = (current_thread_id: number) => {
             const sp = Module.stackSave();
```

Comment:
do we need to assert that current_thread_id is a number, since we're changing the signature of an existing function?

Reply:
no, this is internal method